mssql/README.md

```diff
@@ -0,0 +1,38 @@
+# Spice Demo with SalesLT dataset in MSSQL
```

Comment:
Be consistent with wording. These are all quickstarts, so let's call it MS SQL Server Quickstart.

Reply:
Done